### PR TITLE
rename-screenshot-dir-to-files

### PR DESCRIPTION
**Pull request by sh0ked**
https://github.com/2gis/vmmaster-frontend/pull/38

### CODE REVIEW

File: tests/functional/functional_tests.py [hunk 1]
```diff
@@ -54,78 +54,78 @@ def test_positive_case(self):
...
+#         deffered2.wait()
+#         result1 = deffered1.get()
+#         result2 = deffered2.get()
+#
+#         self.assertEqual(1, result1.testsRun, result1.errors)
+#         self.assertEqual(1, result2.testsRun, result2.errors)
+#         self.assertEqual(0, len(result1.errors), result1.errors)
+#         self.assertEqual(0, len(result2.errors), result2.errors)
+#         self.assertEqual(0, len(result1.failures), result1.failures)
+#         self.assertEqual(0, len(result2.failures), result2.failures)
```
Comment (z00sts) [inline tests/functional/functional_tests.py]: ?

Comment (sh0ked) [inline tests/functional/functional_tests.py]: @z00sts oops... fixed

**Comment by sh0ked** [comment]
!test

**Comment by sh0ked** [comment]
!test

**Comment by sh0ked** [comment]
!test

**Comment by sh0ked** [comment]
!test

**Comment by sh0ked** [comment]
!test

**Pull request closed without merging**